libretro-common/cdrom/cdrom.c

```diff
@@ -301,7 +301,7 @@ static int cdrom_send_command_win32(const libretro_vfs_implementation_file *stre
 
    memcpy(sptd.s.Cdb, cmd, cmd_len);
 
-   ioctl_rv = DeviceIoControl(stream->fh, IOCTL_SCSI_PASS_THROUGH_DIRECT, &sptd,
+   ioctl_rv = DeviceIoControl(stream->parent ? stream->parent->fh : stream->fh, IOCTL_SCSI_PASS_THROUGH_DIRECT, &sptd,
```

Comment:
if the track was opened directly, parent will be null, but the track will hold the file handle.